etc/singularity.conf

```diff
@@ -17,8 +17,30 @@ allow setuid = yes
 # Should we allow users to request the PID namespace?
 allow pid ns = yes
 
-
-# ENABLE OVERLAY: [BOOL]
+# ALLOW USER IMAGE: [BOOL]
+# DEFAULT: yes
+# Should we allow users to run images they own?  If set to 'no', then the image
+# can only be executed through 'protected image mode' (see below).
+#
+# Note that if 'allow user image' is no and 'protected image mode' is none,
+# then no image file may be used.
+allow user image = yes
+
```

Comment:
The permissions examined are those of the image file.  Since they're enforced by Linux, it's based on whenever `singularity` is invoked.